torch/autograd/gradcheck.py

```diff
@@ -59,10 +49,13 @@ def contiguous(input):
     return input
 
 
+# `input` is input to `fn`
+# `target` is the Tensors wrt whom Jacobians are calculated
 def get_numerical_jacobian(fn, input, target, eps=1e-3):
     # To be able to use .view(-1) input must be contiguous
     input = contiguous(input)
-    target = contiguous(target)
+    if not all(t.is_contiguous() for t in iter_tensors(target)):
+        raise RuntimeError("target must only contain contiguous Tensors")
```

Comment:
Why was `contiguous` invalid in here? It computes the derivative numerically so this shouldn't really matter

Reply:
The version here is WIP. But when `target` is not `input`, e.g., in some tests we have `get_numerical_jacobian(module, input, module_params)`, then making them contiguous is a copy, and changing values in it has no effect on the resulting gradients, and thus the numerical result would be 0.

The current gradgradcheck is broken because of this. It only passed because we had the same `contiguous(inputs)` in calculating analytical grads.